ZelBack/src/services/syncthingService.js

```diff
@@ -1723,11 +1723,20 @@ async function debugCpuprof(req, res) {
   const authorized = res ? await verificationHelper.verifyPrivilege('adminandfluxteam', req) : true;
   let response = null;
   if (authorized === true) {
-    response = await performRequest('get', '/rest/debug/cpuprof', undefined, 60000);
+    try {
+      response = await axios.get('/rest/debug/cpuprof', {
+        responseType: 'stream', // Specify response type as stream
+        timeout: 60000,
+      });
+      res.setHeader('Content-Type', 'application/octet-stream');
+      return response.data.pipe(res);
```

Comment:
was this tested?

Reply:
yes, it pipes the response from Syncthing.